docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1272,13 +1272,21 @@
 
                 <tr>
                     <td class="property" id="store-changelog-replication-factor">stores.<span class="store">store-name</span>.changelog.<br>replication.factor</td>
-                    <td class="default">2</td>
+                    <td class="default">stores.changelog.replication.factor.default</td>
```

Comment:
Hi, @jwlent55 thanks! The document I referred to is the web document: http://samza.apache.org/learn/documentation/0.12/jobs/configuration-table.html

The doc is located and updated via ${SAMZA_ROOT}/docs/learn/documentation/versioned/jobs/configuration-table.html 

Hope that helps.